net/redisclient.go

```diff
@@ -249,11 +280,15 @@ func NewRedisRingClient(ro *RedisOptions) *RedisRingClient {
 		r.log = ro.Log
 		r.metricsPrefix = ro.MetricsPrefix
 
-		if ro.AddrUpdater != nil {
+		if ro.AddrUpdater != nil || ro.RedisAddress != "" {
```

Comment:
I think you should provide an AddrUpdater func that is able to do the remote fetch for you.